src/types/Firestore.ts

```diff
@@ -64,13 +63,12 @@ export interface FirestoreData {
   experience: Experience;
   parentVenue?: AnyVenue;
   playaVenues?: Record<string, AnyVenue>; // for the admin playa preview
-  privatechats?: Record<string, PrivateChatMessage>;
```

Comment:
Currently, that map is not used anywhere and I can't imagine that it is going to be needed in the nearest future.

Reply:
The pattern that we have followed for everything else is that both of the types are represented, because the library stores/creates both in redux when its used. So removing this as you have lies about the data that exists there, and doesn't follow the established patterns.